### DISCUSSION TITLE
Copy URL

### DISCUSSION THREAD

**Discussion by shbitsus** [keepassxreboot/keepassxc#6855] [Q&A] [answered]
transitioning from Keepass2 (debian xfce mono).
Is there no way to copy URL?
From Keepass2, it would be Ctrl+Shift+U , or double-click the URL field.
Seems there is no way to place URL on clipboard.
why?:
I have four browsers, about two dozen browser-profiles, and often several open at once.
(to isolate projects, or multiple accounts w/ same cloud provider, put tracking in silos, and to limit possibility of XSS attacks on more valuable accounts)
I need to be able to explicitly open a KeepassXC URL in the browser windows of my choosing.

**Comment by shbitsus** [comment]
Nevermind,  guess I need to find the manual.

**Reply by shbitsus** [reply]
I'm back:  KeyboardShortcuts.adoc says that Ctrl+U copies URL, but instead it opens a new tab in one of my browsers. Seems CTRL+Shift+U does nothing.  Would sure be nice to have the clipboard timer display.  Not only to know if expired yet, but to confirm my 'copy' event, whether initiated by keyboard or mouse, actually happened.

**Reply by droidmonkey** [reply, marked as answer]
Ctrl + U copies the url.  Are you sure you are using KeePassXC and not KeePassX? What version of KeePassXC are you using?

**Reply by shbitsus** [reply]
KeePassXC 2.3.4  from Debian Bullseye repo.

**Reply by droidmonkey** [reply]
We are on 2.6.6, use our appimage, snap, or ppa distribution. 2.3.4 is about 3 years old.

**Reply by shbitsus** [reply]
oops, and I lied.  Actually Bullseye has v2.6.2
This instance is still Buster. I forgot which machine I was on.  I'm in progress of migrating to new hw running Bullseye, but this isn't it.
Thank You !
